presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -313,9 +313,12 @@ void PrestoServer::run() {
       operators::UnsafeRowExchangeSource::createExchangeSource);
 
   velox::dwrf::registerDwrfReaderFactory();
+  velox::dwrf::registerDwrfWriterFactory();
 #ifdef PRESTO_ENABLE_PARQUET
   velox::parquet::registerParquetReaderFactory(
       velox::parquet::ParquetReaderType::NATIVE);
+  velox::parquet::registerParquetWriterFactory(
+      velox::parquet::ParquetReaderType::NATIVE);
```

Comment:
The writer factory as no argument.
You also need to include the header file.

Reply:
You also need to add `#include "velox/dwio/parquet/RegisterParquetWriter.h" // @manual`